dist/index.html

```diff
@@ -20,35 +18,32 @@ <h1 class='title-header'>🌈  Travel Tracker Wishes You a Happy Pride Month 
           <h2 class='welcome-text' id='welcomeText'>Welcome!</h2>
         </div>
         <div tabindex=0 class="right-contents">
-          <!-- <h3 class='dashboard-buttons zoom' role="button" aria-pressed="false" id='homeButton'>Book a New Trip</h3> -->
-          <h4 class='dashboard-buttons zoom' role="button" aria-pressed="false" id='logoutButton'>Logout</h4>
+          <button class='logout zoom rainbow-3' id='logoutButton'>Logout</button>
```

Comment:
the zoom class is what enlarges the things when you hover over!! Curious to see its CSS 🔍